docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -0,0 +1,271 @@
+
+In order to enable Azure instance discovery the `discovery_service.azure` section
+of `teleport.yaml` must include at least one entry:
+
+```yaml
+version: v2
+teleport:
+  join_params:
+    token_name: "/tmp/token"
+    method: token
+  auth_servers:
+  - "teleport.example.com:3080"
+auth_service:
+  enabled: off
+proxy_service:
+  enabled: off
+ssh_service:
+  enabled: off
+discovery_service:
+  enabled: "yes"
+  azure:
+    - types: ["vm"]
+      subscriptions: []
+      resource_groups: []
+      regions: []
+      tags:
+        "env": "prod" # Match virtual machines where tag:env=prod
+```
+
+- Edit the `teleport.auth_servers` key to match your Auth Service or Proxy Service's URI
+  and port.
+- Adjust the keys under `discovery_service.azure` to match your Azure environment,
+  specifically the regions and tags you want to associate with the Discovery
+  Service.
+
+## Step 5/5. [Optional] Customize the default installer script
+
+To customize the default installer script, execute the following command on
+your workstation:
+
+```code
+$ tctl get installer/default-installer > teleport-default-installer.yaml
+```
+
+The resulting `teleport-default-installer.yaml` can the be edited to
+change what gets executed when enrolling discovered Azure instances.
+
+After making the desired changes to the default installer, the
+resource can be updated by executing:
+
+```code
+$ tctl create -f teleport-default-installer.yaml
+```
+
+Multiple `installer` resources can exist and be specified in the
+`azure.install.script_name` section of a `discovery_service.azure` list item in
+`teleport.yaml`:
+
+```yaml
+discovery_service:
+  azure:
+    - types: ["vm"] 
+      tags:
+       - "env": "prod"
+      install: # optional section when default-installer is used.
+        script_name: "default-installer"
+    - types: ["vm"] 
+      tags:
+       - "env": "devel"
+      install:
+        script_name: "devel-installer"
+```
+
+---
+
+The `installer` resource has the following templating options:
+
+- `{{ .MajorVersion }}`: the major version of Teleport to use when
+  installing from the repository.
+- `{{ .PublicProxyAddr }}`: the public address of the Teleport Proxy Service to
+connect to.
+
+These can be used as follows:
+
+```yaml
+kind: installer
+metadata:
+  name: default-installer
+spec:
+  script: |
+    echo {{ .PublicProxyAddr }}
+    echo Teleport-{{ .MajorVersion }}
+version: v1
+```
+
+Which, when retrieved by the Systems Manager document will evaluate to a script
+with the following contents:
+
+```sh
+echo teleport.example.com
+echo Teleport-(=teleport.version=)
+```
+
+The default installer will take the following actions:
+
+- Add an official Teleport repository to supported Linux distributions.
+- Install Teleport via `apt` or `yum`.
+- Generate the Teleport config file and write it to `/etc/teleport.yaml`.
+- Enable and start the Teleport service.
+
+## Troubleshooting
+
+(!docs/pages/includes/server-access/azure-troubleshooting.mdx!)
```

Comment:
I'd also recommend inlining this partial unless we have plans to use it in another guide